.azure-devops/tox.yml

```diff
@@ -0,0 +1,121 @@
+trigger: none
```

Comment:
As a next step we could replace the merge pipeline ....this pipeline is unit tests and linters only right?

Reply:
Correct. This is just unit tests and linters, with parameters to turn on/off individual python and CLI core testing versions.

One thing I plan to adjust eventually (in order to replace the merge pipeline) is to [replace the `pip install -U --target ...` command](https://github.com/Azure/azure-iot-cli-extension/pull/629/files/ebe777c0763d09091ed18f3cb05117548abb3d70#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R64) (as well as the [testSDK step](https://github.com/Azure/azure-iot-cli-extension/pull/629/files/ebe777c0763d09091ed18f3cb05117548abb3d70#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R60)) with a step to install the artifacts from the primary build step in the merge pipeline.

Reply:
Going to merge this as-is, next step will be to replace unit tests in merge pipeline with this tech.